vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -23,19 +23,35 @@
 
   - name: Remove Ironic from source cluster (Ephemeral Cluster is minikube)
     kubernetes.core.k8s:
-      name: capm3-ironic
+      name: "{{ NAMEPREFIX }}-ironic"
       kind: Deployment
       state: absent
       namespace: "{{ IRONIC_NAMESPACE }}"
     when: EPHEMERAL_CLUSTER == "minikube"
 
+  - name: Label BMO CRDs.
+    shell: 'kubectl label --overwrite crds baremetalhosts.metal3.io {{ item }}'
+    with_items:
+       - clusterctl.cluster.x-k8s.io=""
+       - cluster.x-k8s.io/provider="metal3"
```

Comment:
don't we need move label `clusterctl.cluster.x-k8s.io/move` ?

Reply:
No we don't need `clusterctl.cluster.x-k8s.io/move` here at all.

Reply:
have you tested that this was enough?

Reply:
Yes, multiple times!